ui/js/component/formField/view.jsx

```diff
@@ -77,6 +86,13 @@ class FormField extends React.PureComponent {
     });
   }
 
+  clearError() {
```

Comment:
This already exists in `<FormRow />`, and we probably shouldn't have it in 2 places, but I figure it's a 4-line function and we can worry about that later. (`showError` is already there in both components.)